lend/src/components/ActionMMModal/Footer.tsx

```diff
@@ -0,0 +1,328 @@
+                  />
+                  <DetailText>
+                    {formatBigNumber(utilizationRate, utilizationDecimals)}%
+                  </DetailText>
+                </div>
+              </DetailContainer>
+            </Col>
+            <Col xs={3}>
+              <DetailContainer show={show} delay={0.2}>
+                <DetailTitle>Pool Size (USDC)</DetailTitle>
+                <DetailText>
+                  {currency(formatBigNumber(poolSize, 6), {
+                    symbol: "",
+                  }).format()}
+                </DetailText>
+              </DetailContainer>
+            </Col>
+            <Col xs={3}>
+              <DetailContainer show={show} delay={0.3}>
+                <DetailTitle>
+                  {isBorrow ? "Borrow" : "Repay"} Amount
+                </DetailTitle>
+                <DetailText
+                  color={
+                    borrowAmountNum === 0
+                      ? colors.primaryText
+                      : borrowAmountNum > 0
+                      ? colors.green
+                      : colors.red
+                  }
+                >
+                  {currency(
+                    absoluteBorrowAmount
+                      ? formatBigNumber(absoluteBorrowAmount, 6)
+                      : "0.00",
+                    {
+                      symbol: "",
+                    }
+                  ).format()}
+                </DetailText>
+              </DetailContainer>
+            </Col>
+            <Col xs={3}>
+              <DetailContainer show={show} delay={0.4}>
+                <DetailTitle>Borrow APR</DetailTitle>
+                <DetailText>
+                  {currency(apr.toFixed(2), { symbol: "" }).format()}%
+                </DetailText>
+              </DetailContainer>
+            </Col>
+          </DesktopOnly>
+          <MobileOnly>
+            <Col xs={6}>
+              <DetailContainer show={show} delay={0.1}>
+                <DetailTitle>Current Utilization</DetailTitle>
+                <div className="d-flex">
+                  <UtilizationBar
+                    percent={parseFloat(
+                      formatBigNumber(utilizationRate, utilizationDecimals)
+                    )}
+                    width={40}
+                    color="white"
+                  />
+                  <DetailText>
+                    {formatBigNumber(utilizationRate, utilizationDecimals)}%
+                  </DetailText>
+                </div>
+              </DetailContainer>
+            </Col>
+            <Col xs={6}>
+              <DetailContainer show={show} delay={0.2}>
+                <DetailTitle>Borrow APR</DetailTitle>
+                <DetailText>
+                  {currency(apr.toFixed(2), { symbol: "" }).format()}%
+                </DetailText>
+              </DetailContainer>
+            </Col>
+          </MobileOnly>
+        </FooterRow>
+      ) : (
+        <FooterRow>
+          {chainId !== undefined && (
```

Comment:
Can simplify to `!chainId`. You should check if txhash is undefined as well right?